### PR TITLE
Switch to glog

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: deployment/manifest_step.go [hunk 1]
```diff
@@ -2,8 +2,8 @@ package deployment
 
 import (
 	"errors"
-	"log"
 
+	"github.com/golang/glog"
```
Comment (cored) [inline deployment/manifest_step.go]: could you add the alias 'log' to this package?

Comment (macat) [inline deployment/manifest_step.go]: keeping it `glog` makes the code more explicit.

**Comment by poopoothegorilla** [comment]
just a general question. What are our guidelines for when to use `Info`, `Error`, `Warning`, and `Fatal`?

File: store/etcd.go [hunk 2]
```diff
@@ -54,8 +54,7 @@ func (*etcdStore) Values(path string) (values map[string]string) {
 	values = map[string]string{}
 	resp, err := api.Get(context.Background(), path, &etcdclient.GetOptions{Recursive: true})
 	if err != nil {
-		log.Println("Ignoring error getting values:" + path)
-		log.Println(err)
+		glog.Errorf("Ignoring error getting values: %s. Error: %s", path, err.Error())
```
Comment (poopoothegorilla) [inline store/etcd.go]: This is just an example of a line i was semi confused on... I would have chosen `Warn` here since it is ignoring the error

Comment (macat) [inline store/etcd.go]: This should be a warning, fixing it/

**Comment by poopoothegorilla** [comment]
+1

File: deployment/manifest_step.go (hunk 1 above)
Comment (cored) [inline deployment/manifest_step.go]: You already merge it. But I did not get how glog makes the code explicit.

**Pull request merged**